docs/tre-admins/setup-instructions/installing-workspace-service-and-user-resource.md

```diff
@@ -0,0 +1,106 @@
+    ```
+
+    Copy the resulting JSON payload.
+
+1. Navigate to the Swagger UI at `https://<azure_tre_fqdn>/api/docs`. 
+
+1. Log into the Swagger UI by clicking `Authorize`, then `Authorize` again. You will be redirected to the login page.
+
+1. Once logged in. Click `Try it out` on the `POST` `/api/workspace-service-templates` operation.
+
+1. Paste the payload json generated earlier into the `Request body` field, then click `Execute`. Review the server response.
+
+1. To verify registration of the template do `GET` operation on `/api/workspace-service-templates`. The name of the template should now be listed.
+
+## Publish and register a user resource template
+
+The Guacamole workspace service also has user resources, there are the VMs that researchers will deploy. These steps can be repeated for any user resource template.
+
+1. Run:
+
+    ```cmd
+    /workspaces/tre> make register-bundle DIR=./templates/workspace_services/guacamole/user_resources/guacamole-azure-win10vm BUNDLE_TYPE=user_resource
+    ```
+
+    Copy the resulting JSON payload.
+
+1. Navigate to the Swagger UI at `https://<azure_tre_fqdn>/api/docs`. You should already be logged in.
+
+Click `Try it out` on the `POST` `/api/user-resource-templates` operation.
+
+1. In the `service_template_name` field, paste the name of the workspace service template that you registered earlier - `tre-service-guacamole`.
+
+1. Paste the payload json generated earlier into the `Request body` field, then click `Execute`. Review the server response.
+
+1. To verify registration of the template do `GET` operation on `/api/user-resource-templates`. The name of the template should now be listed.
+
+## Creating a workspace service
+
+Now that we have published and registered both workspace service and user resource bundles we can use the workspace API to create a workspace service in our workspace.
+
+1. Navigate to the Swagger UI at `https://<azure_tre_fqdn>/api/<workspace_id>/docs` . Where `<workspace_id>` is the workspace ID of the workspace created in the previous step.
+
+!!! info
+    All routes are auth protected. Click the green **Authorize** button to receive a token for Swagger client.
+
+1. Log into the Swagger UI by clicking `Authorize`, then `Authorize` again. You will be redirected to the login page.
+
+!!! info
+    You need to log in with a user with assigned the WorkspaceOwner role in the app regsitration used when deploying your workspace.
+
+1. Once logged in. Click `Try it out` on the `POST` `/api/workspaces/<workspace_id>/workspace-services` operation.
+
+1. Enter the workspace_id in the `workspace_id` field.
+
+1. Paste the following payload json into the `Request body` field, then click `Execute`. Review the server response.
+
+```json
+{
+  "templateName": "tre-service-guacamole",
+  "properties": {
+    "display_name": "Virtual Desktops",
+    "description": "Create virtual desktops for runnign research workloads"
+  }
+}
```

Comment:
Maybe it is worth adding the whole payload needed for the service, something like this:
```suggestion
{
  "templateName": "tre-service-guacamole",
  "properties": {
    "display_name": "Virtual Desktops",
    "description": "Create virtual desktops for runnign research workloads",
    "tre_id": "<TRE ID>",
    "api_client_id": "<TRE API app registration client id>",
    "aad_tenant_id": "<Tenant id>",
    "openid_client_id": "<Workspace app registration client id>"
  }
}
```

Reply:
Updated.